cloud_edition/flexibility_mode/docs/environments_access.rst

```diff
@@ -102,3 +102,44 @@ If **akeneo**, as an SSH user or as a PIM process, creates files in the SFTP sub
 .. _`specific tasks`: ./partners.html
 .. _`GitHub Help Center`:  https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent
 .. _`Akeneo Help Center`:  https://help.akeneo.com/portal/articles/access-akeneo-flexibility.html?utm_source=akeneo-docs&utm_campaign=flexibility_partner_starterkit
+
+
+Copy data from one instance to other
+************************************
+
+**Scenario:**
+
+    `User` want to transfer a SQL dump file from prod instance toward staging instance:
```

Comment:
Backquotes are not interpreted in this block

Reply:
removed